HARSH/index.html

```diff
@@ -0,0 +1,79 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Calculator</title>
+    <link rel="stylesheet" href="./style.css">
+    <script defer src="./script.js"></script>
+    <style>
+        .none {
+            display: none;
+        }
+    </style>
+</head>
+
+<body>
+    <div id="calculator-box">
+        <div id="calculator">
+            <h3>Calculator</h3>
+            <input type="text" id="result-box">
+            <div id="table1">
+            <table>
+                <tr>
+                    <td><button>(</button></td>
```

Comment:
had you used a bit of bootstrap in css,you could have saved so much time with the buttons:)